tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, config_builder):
     self.compare_custom_predict_fn = (
       config.get('compare_custom_predict_fn')
       if 'compare_custom_predict_fn' in config else None)
+    self.custom_distance_fn = (
+      config.get('custom_distance_fn')
```

Comment:
sorry, did not read the code yet but if this is dict, can't you just say `config.get('custom_distance_fn', None)`?

Reply:
good catch, I just repeated the pattern from other keys. I now changed all other instances of it as well.